src/Leaderboard.ts

```diff
@@ -149,6 +149,9 @@ export default class Leaderboard {
         }`
       )
       .setDescription(`Next update: ${nextUpdate.toLocaleTimeString()}`)
```

Comment:
```suggestion

```

Reply:
I honestly don't know why that is there, I could swear I removed it...